h_transport_materials/property_database/tzm.py

```diff
@@ -0,0 +1,139 @@
+import h_transport_materials as htm
+from h_transport_materials import (
+    Permeability,
+    Diffusivity,
+    Solubility,
+    DissociationCoeff,
+    RecombinationCoeff,
+) 
+import numpy as np
+
+u = htm.ureg
+
+forcey_permeability_d = Permeability(
+    pre_exp=8.32e-8 * u.mol * u.m**-1 * u.Pa**-0.5 * u.s**-1,
+    act_energy=75.5 * u.kJ * u.mol**-1,
+    range=(515 * u.K, 742 * u.K),
+    isotope="D",
+    source="forcey_permeability_1993",
+    note="Equation 8",
+)
+
+tominetti_permeability_h = Permeability(
+    pre_exp=5.53e-5 * 6.90e-4 * u.mol * u.m**-1 * u.Pa**-0.5 * u.s**-1,
+    act_energy=(84.6 - 11.8) * u.kJ * u.mol**-1,
+    range=(673 * u.K, 873 * u.K),
+    isotope="H",
+    source="tominetti_solubility_1990",
+    note="D*Ks",
+)
```

Comment:
In HTM you have the ability to directly multiply a htm.DIffusivity with a htm.Solubility instance.
This is to avoid code duplication (here the value 5.35e-5 is written twice for instance)

Something like:

```python
tominetti_permeability_h = tominetti_diffusivity_h * tominetti_solubility_h
tominetti_permeability_h.source = "tominetti_solubility_1990"
tominetti_permeability_h.isotope = "D"
tominetti_permeability_h.range = (673 * u.K, 873 * u.K)
```